tests/cli_test.py

```diff
@@ -0,0 +1,56 @@
+import sys
+from tests.draw_test import TestBase
+import unittest
+from worldengine import __main__
+from worldengine.cli.main import main
+
+
+class TestCLI(TestBase):
+
+    def setUp(self):
+        super(TestCLI, self).setUp()
+        self.world = "%s/seed_28070.world" % self.tests_data_dir
+
+    def test__main__(self):
```

Comment:
What do you mean?  There is no hard or fast rule on this, but typically in Python when you begin a function or variable/member with _ it is meant to declare it as only usable from within a particular scope. 

Now ipython will 'hide' those, but you can still access them if you really want to and most IDEs will complain (warnings in PyCharm). However in testing, it doesn't matter as you want to test ALL functions anyway.

Not sure if that answers your question or not.